src/docs/markdown/automatic-https.md

```diff
@@ -220,3 +222,84 @@ Before attempting any ACME transactions, Caddy will test the configured storage
 Caddy can obtain and manage wildcard certificates when it is configured to serve a site with a qualifying wildcard name. A site name qualifies for a wildcard if only its left-most domain label is a wildcard. For example, `*.example.com` qualifies, but these do not: `sub.*.example.com`, `foo*.example.com`, `*bar.example.com`, and `*.*.example.com`.
 
 To get a wildcard from Let's Encrypt, you simply need to enable the [DNS challenge](#dns-challenge) and use a wildcard domain in your config. We recommend using wildcards only when you have so many subdomains that you would encounter CA rate limits trying to obtain certificates for them all.
+
+
+
+## Examples
+
+Caddy implicitly uses the HTTPS port (default 443) for your [site addresses](/docs/conventions#network-addresses) that don't specify a port explicitly (which would disable automatic HTTPS). 
+
+Automatic HTTPS can be configured via the Caddyfile with the [`auto_https` global option](/docs/caddyfile/options), or via [per-server JSON configuration](/docs/json/apps/http/servers/automatic_https/). The Caddyfile option can be set to either of the following:
+- `disable_redirects` which disables the implicit HTTP->HTTPS redirect.
+- `off` which disables automatic HTTPS altogether, including the HTTP->HTTPS redirect and automatic enabling of TLS for sites that meet the requirements.
+
+| auto_https        | HTTP        | HTTPS       |
+|-------------------|-------------|-------------|
+| *default*         | redirects   | connects    |
+| disable_redirects | unreachable | connects    |
+| off               | connects    | unreachable |
+
+`unreachable` is the default without additional configuration when using the related `auto_https` setting.
```

Comment:
What does this mean though? Shouldn't this describe how the socket is not opened implicitly?

Also I'm not sure "unreachable" is the right word. Whether a socket is reachable is external and unrelated to the web server: I think you want "does not connect" or something like that. Port 443 (or whatever) might very well be reachable, but not able to connect.

Reply:
Should the table be revised to indicate instead which protocol scheme is implicitly used? In which case should `redirects` be changed/clarified or left as is?

Reply:
> EDIT: Misunderstood how the implicit handling works. There is no implicit HTTP other than the default redirect to HTTPS. This table is not useful then..

I think I agree on this point. The only nuance to this is that Caddy _will_ try to bind to the HTTP port to solve the ACME HTTP challenge, but this doesn't _necessarily_ happen at the same time as starting the server along with the HTTPS ports etc. Caddy still needs the HTTP port if it is to solve the HTTP challenge successfully, so it is used for more than just redirects (unless the HTTP challenge is disabled.)

But that is explained in the docs already.

We can remove the table then.